core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/definition/script/ScriptJsonComponent.java

```diff
@@ -75,11 +76,15 @@ public Script deserialize(JsonParser jsonParser, DeserializationContext deserial
                 versionNumber = versionNode.get(ScriptVersionJsonComponent.Field.NUMBER_KEY.value()).asLong();
                 scriptVersion = new ScriptVersion(
                         new ScriptVersionKey(new ScriptKey(scriptId, versionNumber)),
-                        versionNode.get(ScriptVersionJsonComponent.Field.DESCRIPTION_KEY.value()).asText()
+                        versionNode.get(ScriptVersionJsonComponent.Field.DESCRIPTION_KEY.value()).asText(),
+                        "username",
+                        LocalDateTime.now().toString()
```

Comment:
Where is this used? Isn't this used to load a script from CLI. If so, shouldn't the username be admin?

Reply:
Done